patterns/2-structured/issue-tracker.md

```diff
@@ -50,6 +50,7 @@ development but also during the planning phase of new features:
 ## Known Instances
 
 * Europace AG - See blog post [Issue Use Cases](https://tech.europace.de/post/using-issues-for-asking-questions-and-tracking-work/)
+* Siemens manages work on their InnerSource Platform in a fully transparent way. This includes tracking enhancements and bugs, as well as marketing, communication activities and support requests from users.
```

Comment:
```suggestion
* Siemens manages work on their InnerSource Platform in a fully transparent way. Therefore the work on the InnerSource Platform itself is organized as an InnerSource project. This includes tracking enhancements and bugs, as well as marketing, communication activities and support requests from users.
```

This is the dog fooding part from your [presentation](https://opensource.siemens.io/talks/slides/FOSS-Backstage-2025-Growing-a-Social-Coding-culture.pdf) (slide7) I assume?

Very cool!

I rewrote this a bit, to make it even more clear that the platform itself is run as an InnerSource project.

Reply:
Yes! Great suggestion, thanks!